posixlib/src/main/scala/scala/scalanative/posix/sys/wait.scala

```diff
@@ -0,0 +1,10 @@
+package scala.scalanative.posix.sys
```

Comment:
I did not notice an associated edit to `docs/lib/posixlib.rst` to announce that `wait` is now supported.

My recent PR #1646 added support for resource.h so I had to edit posixlib.rst.  I think/hope I got it right.
Those edits might be useful to you.  The key is in knowing that there are  __two__ places to edit in
the file.     

If it would be useful to you, I could do the wait.h edits, do a sphinx-build to ensure they build (skipping over
known errors), and ship it to you.  I am trying to ease the PR flow, if possible.

Reply:
I see, I don't think I knew that document existed, I'll update this with these functions.

Reply:
it looks to me like sys/wait.h is already listed in there for some reason?

Reply:
I just checked the "latest" (0.4.0-M2) ReadTheDocs web output and posix sys/wait.h
is listed as "N/A" (Not Available? I'll create an separate issue ). 

As this PR implements sys/wait.h, I would expect the right column to
contain a valid link, such as the implemented sys/types.h a few lines above.
(By being concrete I mean to be descriptive, not to give offense.)

Ar my tired eyes missing something?